parsec/stubs/marshmallow/__init__.pyi

```diff
@@ -32,33 +32,41 @@ post_dump: Callable[[T], T]
 class MarshmallowSchema:
```

Comment:
Can be made generic 
but maybe for another PR :thinking:

Reply:
@Volham22 WDYT ?

Reply:
I'll do an other PR for that